utils/zerovec/derive/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "zerovec-derive"
```

Comment:
Question: `zerovec-derive` or `zerovec_derive` ?

Reply:
I typically use dashes and the yoke derive crate does the same